environments/inventory

```diff
@@ -1,11 +1,7 @@
-[all:vars]
-ansible_python_interpreter = /usr/bin/python3
-
 [sirbot]
-pythondev-01
-
-[pyslackers-website]
-website
+pyslackers-01
+pyslackers-02
```

Comment:
I pointed other dns for the second one: https://pyslackers.ovv.wtf/
Feel free to test/experiment on it